src/state/quick-bar-mixin.ts

```diff
@@ -43,6 +48,29 @@ export default <T extends Constructor<HassElement>>(superClass: T) =>
       showQuickBar(this, { commandMode });
     }
 
+    private _createMyLink(e: KeyboardEvent) {
+      if (!this._canOverrideAlphanumericInput(e) || !this.hass) {
+        return;
+      }
+      const targetPath = mainWindow.location.pathname;
+
+      for (const [path, redirect] of Object.entries(
```

Comment:
It's a slug, not a path
```suggestion
      for (const [slug, redirect] of Object.entries(
```